src/Microsoft.DotNet.Interactive/Commands/SerializeInteractiveDocument.cs

```diff
@@ -1,21 +1,19 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.DotNet.Interactive.Notebook;
-
 namespace Microsoft.DotNet.Interactive.Commands
 {
-    public class SerializeNotebook : KernelCommand
+    public class SerializeInteractiveDocument : KernelCommand
```

Comment:
@brettfo and @jonsequitur I have renamed these commands as they deal with our representation of an interactive document and they write it as dib or ipynb. Would like to pick your brain on these two commands. I think we have enough at this point in our object model to close this PR. Contract types are changed anyway (elements vs cells property for example) so I need to also align the typescript code in vscode extension (should be minimal work)

Reply:
Looks good to me.